lib/src/Base/Stat/openturns/CovarianceModel.hxx

```diff
@@ -72,19 +72,13 @@ public:
   virtual Scalar computeStandardRepresentative(const Scalar & s,
       const Scalar & t) const;
 
-  virtual Scalar computeStandardRepresentative(const Point & tau) const;
-
-  virtual Scalar computeStandardRepresentative(const Scalar & tau) const;
```

Comment:
I was unsure about this one; there is no StationaryCovarianceModel_doc, and this page tells that the method with a single argument is designed for stationary covariance models, so this information looks still valid to me.